### ISSUE TITLE
packaging issues

### ISSUE THREAD

**Issue by argriffing** [yiakwy/PyMatrix#1]
Hi I saw that you are making some effort to announce this project on the internet. Here's a list of boring overhead stuff that is required for people to want to consider using your project!
- Licensing. Add a license file to the top level of the directory (BSD, GPL, MIT, whatever). This will let other people use your code!
- Readme. Even if the project does not have real docs, the key features should be listed in the `README.md` file (this file already exists). It should at least briefly list or summarize the features that make it 'powerful and easier than Matlab, R, and NumPy'. This information may be available on your blog but it could be summarized here.
- Installation. There should at least be some kind of `setup.py` at the top level.
- Merge conflicts. Some of the `.py` files in this project have text like `<<<<<<< HEAD` which isn't Python code!
- Versioning. Currently the versions are in separate directories in the top level. I think the usual way is to use git tags for this. http://githowto.com/tagging_versions.

**Comment by yiakwy** [comment]
Yes, thanks for your comments, many many thanks!. @argriffing the right version in the pull request I gave you should be relieved of merging conflicts. I will make things on the right train gradually.
